src/core/layout/scrollable-list/scrollable-list.js

```diff
@@ -4,48 +4,54 @@
  * @author BBC Children's D+E
  * @license Apache-2.0 Apache-2.0
  */
-import { assetKey } from "./scrollable-list-helpers.js";
+import { updatePanelOnFocus, updatePanelOnScroll } from "./scrollable-list-handlers.js";
 import { createGelButton, scaleButton } from "./scrollable-list-buttons.js";
+import * as a11y from "../../accessibility/accessibility-layer.js";
 import fp from "../../../../lib/lodash/fp/fp.js";
 
+const GRID_NAME = "grid";
```

Comment:
Could probably just remove this and rely on "grid" below. 
I get why this is a common pattern - you remove any problems with mis-spelling but it immediately stops all the code being portable.

Reply:
yeah- there's only the smallest semantic gain by making this a `const`. Code style thing I guess, last team preferred to `const` all strings. Will change.

Reply:
44416fa